library/src/lib/calendar/calendar.component.ts

```diff
@@ -100,7 +103,7 @@ export class CalendarComponent implements OnInit, OnChanges, AfterViewChecked {
         'November',
         'December'
     ];
-    weekDays: string[] = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
+    weekDays: string[];
     daysPerMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```

Comment:
what if February is 29 days

Reply:
there is a function `determineDaysInMonth` that determines if Feb has 28 or 29 days